MdePkg/Include/IndustryStandard/IpmiNetFnGroupExtension.h

```diff
@@ -11,6 +11,7 @@
 
 #include <Pi/PiStatusCode.h>
 
+#pragma pack(1)

```

Comment:
I do not see a specification reference in the file header.  Can one be added that contains a clear description of the structure layout to make it clear by byte packing is required?

Reply:
Hi @mdkinney, the ARM Server Base System Architecture (SBSA) specification, version 2.0, is referenced on line 27. Section F clearly defines the offset for each field within the structure. As the structure does not include padding, alignment should be ensured with 1-byte packing.

Should I move the `#pragma pack(1)` after line 27 or include the specification reference in the file header for clarity?

Reply:
Yes.  File header reference is required.  The #pragma pack(1) should only be around structure declarations that are required to be byte package by the specification.

Reply:
Thanks @mdkinney for the clarification. I've updated it.